<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProblemsWidget</class>
 <widget class="QWidget" name="ProblemsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>739</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="problem">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>671</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="choiceA">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QTextBrowser" name="choiceC">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QPushButton" name="previous">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>previous</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="choiceB">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QTextBrowser" name="choiceD">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QPushButton" name="send">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>410</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QLabel" name="User1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>540</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>User1:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="usr1port">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>540</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="usr2ip">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>600</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="User2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>600</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>User2:</string>
   </property>
  </widget>
  <widget class="QLabel" name="User3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>660</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>User3:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="usr1ip">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>540</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>510</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IP Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="Port">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>510</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="usr1correct">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>540</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Correct">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>510</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Correct</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="usr1total">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>540</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Total">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>510</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Total</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="usr1ratio">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>540</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Ratio">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>510</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Ratio</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="usr3ip">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>660</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="usr2port">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>600</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="usr3port">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>660</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="usr2correct">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>600</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="usr3correct">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>660</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="usr2total">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>600</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="usr3total">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>660</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="usr2ratio">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>600</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="usr3ratio">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>660</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
